src/services/mongoose/accounts.ts

```diff
@@ -19,9 +19,20 @@ const projection = {
 }
 
 export const AccountsRepository = (): IAccountsRepository => {
+  const listUnlockedAccounts = async (): Promise<Account[] | RepositoryError> => {
+    try {
+      const result: UserType[] /* UserType actually not correct with {projection} */ =
+        await User.find({ status: AccountStatus.Active }, projection)
+      if (result.length === 0) return new CouldNotFindError()
```

Comment:
This isn't really an error or? When calling a `listXxxCriteria` function then an empty array is a valid return. It would signify there were no records that match the criteria.
```
if (result.length === 0) return []
```